composer.json

```diff
@@ -9,7 +9,7 @@
         }
     ],
     "require": {
-        "php": ">= 8.1",
+        "php": "~8.4.0",
```

Comment:
I would change this to "=>8.4". Otherwise either everyone immediately updates to 8.5 once it comes out, or nobody does. Package managers do not all release new versions on the same day.